test/client/getOrderbook.ts

```diff
@@ -72,142 +55,118 @@ function normalRippledResponse(
   throw new Error('unexpected end')
 }
 
-// function xrpRippledResponse(
-//   request: BookOffersRequest,
-// ): Record<string, unknown> {
-//   if (request.taker_pays.issuer === 'rp8rJYTpodf8qbSCHVTNacf8nSW8mRakFw') {
-//     return rippled.book_offers.xrp_usd
-//   }
-//   if (request.taker_gets.issuer === 'rp8rJYTpodf8qbSCHVTNacf8nSW8mRakFw') {
-//     return rippled.book_offers.usd_xrp
-//   }
-//   throw new Error('unexpected end')
-// }
+function xrpRippledResponse(
+  request: BookOffersRequest,
+): Record<string, unknown> {
+  if (request.taker_pays.issuer === 'rp8rJYTpodf8qbSCHVTNacf8nSW8mRakFw') {
```

Comment:
Shouldn't this be `takerPays`?

Reply:
No. The input is the well formed request and that has parameters as the rippled ones!

Reply:
It should be corrected then, so that both aligned

Reply:
This wasn't resolved - this should be `request.takerPays.issuer`, since the request takes a `takerPays` param. Same with `takerGets`

Reply:
Now I'm really confused about how this still passes the tests

Reply:
oh nvm, ignore - this is the `rippled` params.